ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarDemoController.swift

```diff
@@ -0,0 +1,646 @@
+            UIColor(red: 0.18, green: 0.45, blue: 0.96, alpha: 1).cgColor,
+            UIColor(red: 0.36, green: 0.80, blue: 0.98, alpha: 1).cgColor,
+            UIColor(red: 0.45, green: 0.72, blue: 0.22, alpha: 1).cgColor,
+            UIColor(red: 0.97, green: 0.78, blue: 0.27, alpha: 1).cgColor,
+            UIColor(red: 0.94, green: 0.52, blue: 0.20, alpha: 1).cgColor,
+            UIColor(red: 0.92, green: 0.26, blue: 0.16, alpha: 1).cgColor,
+            UIColor(red: 0.45, green: 0.29, blue: 0.79, alpha: 1).cgColor]
+
+        let colorfulGradient = CAGradientLayer()
+        let size = CGSize(width: 76, height: 76)
+        colorfulGradient.frame = CGRect(x: 0, y: 0, width: size.width, height: size.height)
+        colorfulGradient.colors = gradientColors
+        colorfulGradient.startPoint = CGPoint(x: 0.5, y: 0.5)
+        colorfulGradient.endPoint = CGPoint(x: 0.5, y: 0)
+        colorfulGradient.type = .conic
+
+        var customBorderImage: UIImage?
+        UIGraphicsBeginImageContext(size)
+        if let context = UIGraphicsGetCurrentContext() {
+            colorfulGradient.render(in: context)
+            customBorderImage = UIGraphicsGetImageFromCurrentImageContext()
+        }
+        UIGraphicsEndImageContext()
+
+        return customBorderImage
+    }
+
+    private var allDemoAvatarsCombined: [MSFAvatar] = []
+
+    private var demoAvatarsBySection: [AvatarDemoSections: [AvatarDemoRows: MSFAvatar]] = [:]
+
+    private func initDemoAvatars() {
+        AvatarDemoSections.allCases.filter({ section in
+            return section.isDemoSection
+        }).forEach { section in
+            var avatarsForCurrentSection: [AvatarDemoRows: MSFAvatar] = [:]
+
+            AvatarDemoRows.allCases.filter({ row in
+                return row.isDemoRow
+            }).forEach { row in
+                let avatar = MSFAvatar(style: row.avatarStyle,
+                                       size: section.avatarSize)
+                let avatarState = avatar.state
+                avatarState.primaryText = row.avatarPrimaryText
+                avatarState.image = row.avatarImage
+
+                avatarsForCurrentSection.updateValue(avatar, forKey: row)
+                allDemoAvatarsCombined.append(avatar)
+            }
+
+            demoAvatarsBySection.updateValue(avatarsForCurrentSection, forKey: section)
+        }
+    }
+
+    private func nextPresence() -> MSFAvatarPresence {
+        var presence = presenceIterator.next()
+
+        if presence == nil {
+            presenceIterator = MSFAvatarPresence.allCases.makeIterator()
+            presence = presenceIterator.next()
+        }
+
+        if presence! ==  .none {
+            presence = presenceIterator.next()
+        }
+
+        return presence!
+    }
+
+    @objc private func toggleEnablePointerInteraction(switchView: UISwitch) {
+        isPointerInteractionEnabled = switchView.isOn
+    }
+
+    @objc private func toggleShowPresence(switchView: UISwitch) {
```

Comment:
Doesn't seem like any of these private `toggle*` functions are being used anymore

Reply:
Thanks for catching! Yes, they were all replaced by the updateSetting func.